ui/cypress/integration/node/nodetabs.spec.js

```diff
@@ -0,0 +1,121 @@
+beforeEach(() => {
+  cy.setupMocks();
+  cy.login();
+});
+
+// Navigation tests
+describe('Overview tab in master-0 node page', () => {
```

Comment:
That's not a generic "context" name - what you are testing is rather `Node page overview tab` (no need to specify a node name)